### PR TITLE
[4.0] qn codestyle

### PR DESCRIPTION
**Pull request by brianteeman**
All new code is being forced to follow the codestyle and not to use $q or $qn

This pr updates all the remaining instances in core

code review

### CODE REVIEW

File: plugins/system/debug/debug.php [hunk 1]
```diff
@@ -1445,11 +1445,11 @@ public function onAfterDisconnect(ConnectionEvent $event)
 
 					if ($this->sqlShowProfiles)
 					{
-						foreach ($this->sqlShowProfiles as $qn)
+						foreach ($this->sqlShowProfiles as $quoteName)
```
Comment (zero-24) [inline plugins/system/debug/debug.php]: this and the following does not seam to be expected to be replaced to `quoteName`?

**Comment by zero-24** [comment]
Please also take a look into the drone issues now complain about too long lines?http://ci.joomla.org/joomla/joomla-cms/9419/8

Beyond that two points this looks fine for me! Thanks!

File: plugins/system/debug/debug.php (hunk 1 above)
Comment (brianteeman) [reply plugins/system/debug/debug.php]: no idea myself why it would not be updated

Comment (mbabker) [reply plugins/system/debug/debug.php]: TBH this should've never been named `$qn` anyway as it hardly describes what data is contained within the variable, and renaming it `$quoteName` just makes that fact even worse.

Comment (brianteeman) [reply plugins/system/debug/debug.php]: suggestions?

Comment (brianteeman) [reply plugins/system/debug/debug.php]: i reverted the change for this pr.

**Comment by wilsonge** [comment]
Thanks!

**Comment by brianteeman** [comment]
With small steps we make big progress

**Pull request merged**